src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityResourceIntTest.kt

```diff
@@ -718,12 +718,12 @@ class ActivityResourceIntTest : IntegrationTestBase() {
               courseScheduleRule()
             }
           }
-          offenderBooking = offender(nomsId = "A1234TT") {
-            booking(agencyLocationId = "LEI") {
+          newOffender(nomsId = "A1234TT") {
+            offenderBooking = booking(agencyLocationId = "LEI") {
               incentive(iepLevelCode = "STD")
               courseAllocation(courseActivity)
             }
-          }.latestBooking()
+          }
```

Comment:
All calls to `latestBooking()` have been removed - the booking is returned from the DSL now.